code/modules/mob/living/silicon/ai/vox_sounds.dm

```diff
@@ -31,10 +31,7 @@ GLOBAL_LIST_INIT(vox_sounds, list("abduction" = 'sound/vox_fem/abduction.ogg',
 "aft" = 'sound/vox_fem/aft.ogg',
 "agent" = 'sound/vox_fem/agent.ogg',
 "ai" = 'sound/vox_fem/ai.ogg',
-"airlock" = 'sound/vox_fem/airlock.ogg',
 "air" = 'sound/vox_fem/air.ogg',
-"alarm" = 'sound/vox_fem/alarm.ogg',
-"alert" = 'sound/vox_fem/alert.ogg',
```

Comment:
Can you put these back in ABC order? Should be easy to do it automatically in VScode

Reply:
i code in notepad++ bro

Reply:
did